### PR TITLE
Update the copy within the compedium for Divine Font's remaster changes

### PR DESCRIPTION
**Pull request by Minniehajj**
This PR updates the Divine Font copy within compendium entries for the updates that have been made for the Cleric's Divine Font ability

### CODE REVIEW

File: packs/paizo-pregens/ufi.json [hunk 1]
```diff
@@ -3480,4 +3480,4 @@
         }
     },
     "type": "character"
-}
+}
```
Comment (stwlam) [inline packs/paizo-pregens/ufi.json:3483]:
```suggestion
}

```

File: packs/paizo-pregens/lavanna-saltspray.json [hunk 2]
```diff
@@ -5576,4 +5576,4 @@
         }
     },
     "type": "character"
-}
+}
```
Comment (stwlam) [inline packs/paizo-pregens/lavanna-saltspray.json:5579]:
```suggestion
}

```

File: packs/paizo-pregens/kellsti.json [hunk 3]
```diff
@@ -4610,4 +4610,4 @@
         }
     },
     "type": "character"
-}
+}
```
Comment (stwlam) [inline packs/paizo-pregens/kellsti.json:4613]:
```suggestion
}

```

File: packs/paizo-pregens/eteleon.json [hunk 4]
```diff
@@ -5742,4 +5742,4 @@
         }
     },
     "type": "character"
-}
+}
```
Comment (stwlam) [inline packs/paizo-pregens/eteleon.json:5745]:
```suggestion
}

```

File: packs/kingmaker-bestiary/tristian-level-10.json [hunk 5]
```diff
@@ -6385,4 +6385,4 @@
         }
     },
     "type": "character"
-}
+}
```
Comment (stwlam) [inline packs/kingmaker-bestiary/tristian-level-10.json:6388]:
```suggestion
}

```

File: packs/kingmaker-bestiary/tristian-level-1.json [hunk 6]
```diff
@@ -3314,4 +3314,4 @@
         }
     },
     "type": "character"
-}
+}
```
Comment (stwlam) [inline packs/kingmaker-bestiary/tristian-level-1.json:3317]:
```suggestion
}

```

File: packs/kingmaker-bestiary/sepoko.json [hunk 7]
```diff
@@ -3075,4 +3075,4 @@
         }
     },
     "type": "npc"
-}
+}
```
Comment (stwlam) [inline packs/kingmaker-bestiary/sepoko.json:3078]:
```suggestion
}

```

File: packs/journals/classes.json [hunk 8]
```diff
@@ -491,4 +491,4 @@
             }
         }
     ]
-}
+}
```
Comment (stwlam) [inline packs/journals/classes.json:494]:
```suggestion
}

```

**Comment by TikaelSol** [comment]
The class journal changes are fine but the pregen changes may need some thought about if the change should be retroactive on OGL NPCs and pregens like that. Likely fine but we aren't otherwise rebuilding them beyond migrating to make them still function.

**Comment by TikaelSol** [comment]
Also change the title to imperative case.

**Comment by Minniehajj** [comment]
@TikaelSol I was conflicted on that as well. For now I'll remove the pregen changes to keep things consistent!

**Review by Minniehajj — COMMENTED** [review]
only addded the jounral changes

**Pull request closed without merging**